README.md

```diff
@@ -1 +1,86 @@
-Put all your good documentation here. 
+Welcome to Risknet! This is a downloadable Pip package where you can access and run an XGBoost pipeline.
+
+# Layout
+- src/risknet
+   - config: holds `conf.yaml`
+      - `conf.yaml`: helps with setup
+      - `handlers.py`
+   - data: empty folder where user can store FM dataset
+   - jobs
+      - `__init__.py`
```

Comment:
no need to document __init__.py, those are implicit

Reply:
Removed documentation on `__init__.py` files for README.